.travis.yml

```diff
@@ -6,7 +6,7 @@ language: python
 
 python:
   - '3.6'
-  - '3.7'
+  - '3.7.3'
```

Comment:
Setting travis build to python v3.7.3, the default python v3.7.1 is failing to build pyyaml.